packages/phpstan-rules/tests/Rules/DeadCode/UnusedPublicClassConstRule/Fixture/LocallyUsedPublicConstantByName.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Symplify\PHPStanRules\Tests\Rules\DeadCode\UnusedPublicClassConstRule\Fixture;
+
+final class LocallyUsedPublicConstantByName
+{
+    public const LOCALLY_ONLY_NAMED = 'public is not correct';
+
+    public function run()
+    {
+        return LocallyUsedPublicConstantByName::LOCALLY_ONLY_NAMED;
```

Comment:
difference between this test and the existing one: `self::` vs.`LocallyUsedPublicConstantByName::` referenced constant within the same class